src/serial_reader.cpp

```diff
@@ -76,6 +89,16 @@ SerialReader::SerialReader(const std::string &serial_port, const int &num_values
     rt_thread_.create_realtime_thread(&SerialReader::loop, this);
 }
 
+bool SerialReader::open_port(const std::string &port)
+{
+    fd_ = open(port.c_str(), O_RDWR | O_NOCTTY | O_NDELAY);
+    if (fd_ != -1)
+    {
+        log_->info("Opened serial port {}", port);
```

Comment:
I do not know the output of this but quick trick that helps a lot,
the message name should contain the name of class and the name of the function it's in:
```
log_->info("SerialReader::open_port(): Opened serial port {}", port);
```

Reply:
With default settings the output looks like this:
```
[2022-12-01 07:42:14.099] [SerialReader] [debug] Try to open serial port '/dev/ttyACM'
```
"SerialReader" is the name of the logger in this case, which I set to be the same as the class, so that part of information is already included.
I'll check if it is possible to use some custom formatting that also shows the function name.

In general, I have not yet worked out a good way how to configure loggers (change level, set custom format, etc.) as a user, especially when using the Python bindings.  I'll look a bit more into this.